Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,23 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import "./Header.css";
+
+const HeaderTitle = () => {
+  return (
+    <div className="header-tittle-container">
+      <div className="top-tittle">
```

Comment:
This part is the header title. The bottom-title section should go in the header content component.